javascripts/AppConfig.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+
+app.run(function(FIREBASE_CONFIG){
+	firebase.initializeApp(FIREBASE_CONFIG);
+});
+
+app.config(function($routeProvider){
+	$routeProvider
+		.when("/login", {
+			templateUrl: 'partials/login.html',  
+			controller: 'LoginCtrl' 
+		})
+		.when("/view", {
```

Comment:
This route should be `/contacts/view` instead of `/view` (same with the 2 below).

Reply:
Corrections made.